src/app/organizations/organizations/organizations.component.scss

```diff
@@ -4,14 +4,56 @@
   @extend .pr-5;
 }
 
-.mat-card-title {
-  white-space: nowrap;
+.h5,
+.org-topic {
+  color: #151c3c;
+}
+
+.org-topic {
+  font-weight: 300;
+  line-height: normal;
 }
 
 .orgStar {
-  line-height: 1.1em;
+  font-size: 1.5rem;
+  color: #3f51b5;
+  height: auto;
+  padding: 0 !important;
+}
+
+.info.span {
+  font-size: 10px;
+}
+
+.orgLogo {
+  height: 50px;
+  object-fit: scale-down;
+}
+
+.info-icon {
+  font-size: 1.5rem;
+  height: auto;
+  color: #397a84;
+  padding: 0 !important;
+}
+
+.info {
+  font-size: 12px;
+  line-height: 2rem;
+}
+
+hr {
+  margin-bottom: 2rem;
```

Comment:
we already have classes for this. check the bootstrap4.scss

Reply:
Thanks I forgot about it, but now the funny thing is bootstrap4 doesn't include anything `2rem`

Reply:
find one that's similar